aosp_diff/preliminary/development/python-packages/0001-Adding-mako-module-to-python-development-packages.patch

```diff
@@ -0,0 +1,8206 @@
+From eeb07e7b48b8c28f0c683a5b9d77fd57cae0ee0b Mon Sep 17 00:00:00 2001
+From: yaravapa <yasoda.aravapalli@intel.com>
+Date: Tue, 19 Jan 2021 16:35:46 +0530
+Subject: [PATCH] Adding mako module to python development packages
+
+This is required as part of enabling path restrictions
+in Android 11.
+Python scripts in hardware/intel/mesa3d are dependent on
```

Comment:
Mesa path should be hardware/intel/external/mesa3d-intel.

Reply:
Updated the patch commit message with correct Mesa path hardware/intel/external/mesa3d-intel.